.rubocop_todo.yml

```diff
@@ -67,3 +67,27 @@ RSpec/AnyInstance:
 # URISchemes: http, https
 Metrics/LineLength:
   Max: 279
+
+Rails/HasManyOrHasOneDependent:
```

Comment:
Hey, thanks for the pr. 
What do you think about moving disabled cops (`Rails/HasManyOrHasOneDependent` and `Rails/SkipsModelValidations`) to the `.rubocop.yml`. They seem more like a configuration, not a todo.

Reply:
Yeah configuration should be placed in `.rubocop.yml`

Reply:
Personally I would resolve the offenses. Both of those cops are highlighting API usage where being explicit is safer. I started working on both but didn't have time to finish them.